### ISSUE TITLE
How to combine two models to a single pb file?

### ISSUE THREAD

**Issue by tongvantruong** [davidsandberg/facenet#1150]
Dear,

My goal is adding more dataset to the pre-trained model to increase the accuracy.

1. I freezed the graph to get the 1st model named **pre-trained-model.pb**
2. Then I trained another model based on my own images followed [this topic](https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1)  named **new-model.pb**

How can I merge these 2 model into one **final-model.pb**? or can I feed the 1st model to the second model to produce the final single model (pb)?

**Comment by xioto** [comment]
This may be of help :
https://luuil.github.io/2018/09/15/Merging-Models-for-TensorFlow-Serving/
